### ISSUE TITLE
Toggle Series Button Present Below the Graph is not Announcing Position.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13936]
### Repro steps:
1. Open URL: https://www.highcharts.com/demo/column-basic in the Edge browser
2. Navigate to ‘Basic Column’ under ‘Column and Bar Charts’ in the left navigation.
3. Enable NVDA.
4. Navigate to toggle present below the graph and observe the issue.

### Actual Result:
NVDA doesn't announce the position for the toggle button.
NVDA is just announcing as - 'Toggle series visibility, Monthly Average Rainfall region list with 4 item Show Tokyo toggle button pressed.

### Expected Result:
NVDA should announce the position for the toggle button.
NVDA should announce as - 'Toggle series visibility chart region list, , Monthly Average Rainfall region list with 4 item Show Tokyo toggle button pressed 1 of 4. 

### Screenshot:
![image](https://user-images.githubusercontent.com/93735775/180326161-483b9f6d-087a-43fd-96d5-467523a51af1.png)

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by feerrenrut** [comment]
We're going to close this because we can't progress it as it is.

**Comment by msftedad** [comment]
Hi Team, 
Please refer the URL: https://www.highcharts.com/demo/column-basic you can get the code pen on activating 'Edit in Codepen' in above link. Please refer the screenshot below.
![codepen](https://github.com/nvaccess/nvda/assets/93735775/a5cc3fc5-6190-4c1c-866d-22ab123d5b00)

Please can you reopen the issue as it is still reproducible. Thanks

**Comment by seanbudd** [comment]
as mentioned before, we need a **minimal** sample to be used to reproduce the problem
